src/nccl_ofi_rdma.c

```diff
@@ -5889,7 +5873,7 @@ static void get_hints(struct fi_info *hints)
 
 	hints->ep_attr->type = FI_EP_RDM;
 
-	hints->domain_attr->mr_mode = FI_MR_LOCAL | FI_MR_HMEM | FI_MR_ENDPOINT | FI_MR_VIRT_ADDR |
+	hints->domain_attr->mr_mode = FI_MR_LOCAL | FI_MR_HMEM | FI_MR_VIRT_ADDR |
```

Comment:
Is that because we have to register bounce buffers on both control and data rails or something else?